src/element.ts

```diff
@@ -340,7 +340,7 @@ export class Element {
 
   /** Get the boundingBox. */
   getBoundingBox(): BoundingBox {
-    return new BoundingBox(0, -this.textMetrics.actualBoundingBoxAscent, this.width, this.height);
+    return new BoundingBox(this.x, this.y - this.textMetrics.actualBoundingBoxAscent, this.width, this.height);
```

Comment:
This looks like the core bug fix. Was the previous implementation just incorrect? Or are we changing the definition of the bounding box here?

Reply:
It was incorrect, it was not taking into account the origin of the element.